### ISSUE TITLE
How to connect two containers that are using this for their network?

### ISSUE THREAD

**Issue by cradcore** [qdm12/private-internet-access-docker#33]
If I have the PIA Docker running, and startup two other containers, A and B, both of which are using --net=container:pia, is there a way for A to communicate with B? 

If A and B are using --net=host, they are able to communicate with each other using the machine's local IP, or if A is using PIA and B is using the host network, B is still able to connect to A. However, if they are both using PIA, they can no longer see each other. Is there a way to resolve this?

In other words, is there a way for a container using this image for its network to communicate with other services that are running on my local network?

Thanks for the awesome software!

**Comment by qdm12** [comment]
Hi there,

A and B should be as if they are on the same machine (PIA container) I think because of the `--net="container:pia"`. So try with `127.0.0.1:<port of A or B>`? I did not test it so this is not a 100% sure answer 😄

**Comment by cradcore** [comment]
Yeah, I tried that and unfortunately it doesn't work. I then created a bash shell into the container and tried all 3 IPs assigned to them but to no avail

**Comment by cradcore** [comment]
I was able to get it to work. By creating a new network and changing --net=bridge to --net=pia_network when starting the PIA docker, I was able to connect using 127.0.0.1! Might be helpful to add something like this to the wiki :) Thanks!
